app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+end
```

Comment:
- [ ]  As I said in post model, here should a method of updating likes counter  than putting it in Post class model and make sure it is working as it is supposed to.